### DISCUSSION TITLE
Change icon/logo

### DISCUSSION THREAD

**Discussion by dystechnic** [Daveiano/weewx-wdc#201] [Q&A]
Hi,

I'm setting this skin up for an observatory and we would like to change the logo on the pages to the logo of our observatory. But I can't seem to get it to work.
I made 4 versions of the logo corresponding to the sizes op the icon-192x192.png, etc. I've put them in the same dir as the icon files and changed the line in html-head. inc to <link rel="apple-touch-icon" href="$get_base_path(path='halley-192x192.png')" />.
After removing all files from the www-root and adding the png's to the copy_once line in skin.conf I regenerated all pages. But still the original logo is in the top right corner.

What am I doing wrong? 

Thanks.

**Comment by Daveiano** [comment]
Hi!

You are referring to the logo on the top left in the header (the blue one), right?

There is currently no built-in way to change the logo, it is simply included in the template as an SVG: https://github.com/Daveiano/weewx-wdc/blob/3.x/skins/weewx-wdc/includes/ui-shell.inc#L23

You could add your logo as an SVG and include that, this should work, I guess. You could also replace the provided icon with your own (https://github.com/Daveiano/weewx-wdc/blob/3.x/skins/weewx-wdc/includes/icons/wdc.svg).

I did not think that a user would probably want to change the icon, to be honest. I will include an easy method of replacing the icon in the next release - because this sounds like a useful addition to the skin.

Thanks for reaching out!

**Comment by dystechnic** [comment]
Hi,

We are creating an educational touchpanel for children who visit our observatory.The data from our weather sensors is one of the things we will be presenting to them using weewx and your skin. And because there is some sort of internal standard of presenting things I was fiddling with colors and the logo. 
I will wait for the next release, that sounds easier ;-)

Thank you for your quick reaction. I have more wishes that I am trying to realize so maybe I will come back with some other ideas ;-)

Best Regards!

**Comment by Daveiano** [comment]
> We are creating an educational touchpanel for children who visit our observatory. The data from our weather sensors is one of the things we will be presenting to them using weewx and your skin.

Wow, great! I appreciate this!

Please feel free to comment or create an issue if you run into some problems. I am happy to implement new improvements to the skin, which benefit all users. 

Please be patient, I am currently under a heavy workload, so the next release won't be available before November, I guess.
